src/systems/log/LogPlayback.cc

```diff
@@ -624,6 +624,8 @@ void LogPlayback::Update(const UpdateInfo &_info, EntityComponentManager &_ecm)
 
   // flag changed entity poses as periodically changed based on
   // the latest LogPlaybackPrivate::Parse results
+  // TODO(jenn) When merge forward to Fortress, remove pose
```

Comment:
Could we introduce code that makes what the comments says automatically? I mean: check for major version, if it is 5 or greater skip the code.

Reply:
My plan was to immediately start forward porting this and removing the pose info for Fortress once this is merged. Should I still implement this?

Reply:
if that is the case, I think we don't need it.